smriprep/workflows/base.py

```diff
@@ -114,6 +116,8 @@ def init_smriprep_wf(
     work_dir : str
         Directory in which to store workflow execution state and
         temporary files
+    bids_filters : dict
+        For BIDSDataGrabber output_query
```

Comment:
Clearer documentation, possibly a link to somewhere we describe the syntax, would be good.